### PR TITLE
added Auto signal 

### PR DESCRIPTION
**Pull request by UmangSharmaMobis**

### CODE REVIEW

File: spec/include/PowerOptimize.vspec [hunk 1]
```diff
@@ -108,3 +108,14 @@ PowerOptimizeLevel:
...
                Level 0 indicates that all functionality is enabled, no power optimization enabled.
                Level 10 indicates most aggressive power optimization mode, only essential functionality enabled.
+
+isAutoPowerOptimize:
+  datatype: boolean
+  type: actuator
+  description: Auto Power Optimization Flag When set to 'true',
+               the system enables automatic power optimization,
+               dynamically adjusting the power optimization level based on runtime conditions
+               or features managed by the OEM. When set to 'false',
```
Comment (erikbosch) [inline spec/include/PowerOptimize.vspec:118]: There is a trailing blank on this line causing CI to fail

**Comment by erikbosch** [comment]
For this one to be considered for merging, please fix the errors reported from Continuous Integration

**Comment by UmangSharmaMobis** [comment]
I was on leave without access to my system, I've now successfully resolved the errors reported from Continuous Integration.

File: spec/include/PowerOptimize.vspec [hunk 2]
```diff
@@ -108,3 +108,12 @@ PowerOptimizeLevel:
                A higher number indicates more aggressive power optimization.
                Level 0 indicates that all functionality is enabled, no power optimization enabled.
                Level 10 indicates most aggressive power optimization mode, only essential functionality enabled.
+
+isAutoPowerOptimize:
```
Comment (erikbosch) [inline spec/include/PowerOptimize.vspec:112]: I would prefer capital "I", i.e. `IsAutoPowerOptimize` as all other VSS signals starts with capital I.

Comment (UmangSharmaMobis) [reply spec/include/PowerOptimize.vspec]: Updated

**Comment by erikbosch** [comment]
Meeting notes: merge

**Review by erikbosch — APPROVED** [review]

**Pull request merged**